Remove hard-coded links, formatting edits (#5738)
SumoLogic/sumologic-documentation: docs/integrations/sumo-apps/opentelemetry-collector-insights.md

Before configuring the OTEL Collector integration, ensure you have the following prerequisites in place:

> 1. **Sumo Logic OTLP Source**. You need to create an OTLP source in your Sumo Logic hosted collector. The OTLP source will provide the endpoint URL that the OTEL Collector will use to send telemetry data. Learn more at [Creating a Sumo Logic OTLP Source](/docs/send-data/hosted-collectors/http-source/otlp/).

Reviewer: ```suggestion
Before configuring the OTEL Collector integration, you need to create an OTLP source in your Sumo Logic hosted collector. The OTLP source will provide the endpoint URL that the OTEL Collector will use to send telemetry data. Learn more at [Creating a Sumo Logic OTLP Source](/docs/send-data/hosted-collectors/http-source/otlp/).
```

This sample query monitors queue health from the **Exporter Queue Health** panel.

```sql

> sumo.datasource=otel_collector metric=otelcol_exporter_queue_size deployment.environment=*

Author: ```suggestion
sumo.datasource=otel_collector metric=otelcol_exporter_queue_size deployment.environment=* 
```

Watch for logs containing "Dropping data because sending_queue is full" and monitor failed enqueue metrics:
- `otelcol_exporter_enqueue_failed_spans`

> - `otelcol_exporter_enqueue_failed_metric_points`

Author: ```suggestion
- `otelcol_exporter_enqueue_failed_metric_points` 
```
